shared/testing/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id("krypt.android.library")
+}
+
+android {
+    namespace = "ir.mehdiyari.krypt.testing"
+
+    defaultConfig {
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+}
+
+
+dependencies {
+    implementation("androidx.core:core-ktx:1.8.0")
+    implementation("androidx.appcompat:appcompat:1.6.1")
+    implementation("com.google.android.material:material:1.9.0")
```

Comment:
Why do we need these dependencies? If we need them please use the version catalog like the rest of the gradle files.